blocks/dealer-locator/dealer-locator.js

```diff
@@ -19,6 +19,7 @@ export default async function decorate(block) {
   // add the zip code to the input search, if it is present
   const zipCode = hasZipLocation ? escapeHTML(searchParams.get('l')) : null;
   const datasource = block.textContent.trim();
+  if (!GOOGLE_API_KEY) return;
```

Comment:
Why this?

Reply:
now I saw that is in the old DL, I'll remove it

Reply:
UPDATE: I add an empty string in case the api key is falsy